Assets/Scripts/Audio/Sound/GlobalSoundPlayer.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+
+namespace Audio {
+    namespace Sound {
+
+        public class GlobalSoundPlayer : MonoBehaviour {
+
+        }
```

Comment:
May remove this at some point, but it might also be nice to have some sounds that can be played via the eventChannel like so:

```
eventChannel.OnPlaySound.Invoke("DonsAwesomeDadJoke");
```